### PR TITLE
Adds Officer Britsky

### PR DESCRIPTION
**Pull request by Arthurdentist**
He's ready to check your loicenses

https://www.youtube.com/watch?v=BoQkPLYIEYs

things to do

- Add a list of things that Britsky will arrest you for - DONE
- Make a way to construct Britsky - DONE
- Custom arrest text - DONE

I couldn't make any sound files I was satisfied with but anyone else is welcome to do that in a later PR.

This PR should be done, unless there is anything else I missed. 

:cl:

- Added officer britsky, who can be constructed in robotics and will arrest people for carrying unlawful items without a loicense.

### CODE REVIEW

**Comment by Hoshi-chan** [comment]
Make his "weapon permit" setting include all sorts of non-weapons, like engineering tools, kitchen utensils, and surgical tools.

Also, add a crate to cargo that includes an officer britsky, as well as appropriate uniforms for sec to do a department-wide gimmick.

Okay sorry, I'll stop being an idea guy now.

**Comment by Kurfursten** [comment]
I feel like he should stun anyone standing too close to an entertainment monitor for lacking a TV license.

**Comment by soyconsumer** [comment]
shitcurity

File: code/game/machinery/bots/secbot.dm [hunk 1]
```diff
@@ -1115,6 +1122,32 @@ Auto Patrol: []"},
...
+		/obj/item/weapon/gun,
+		/obj/item/weapon/melee,
+		/obj/item/toy/,
+		/obj/item/ashtray,
+		/obj/item/candle,
+		/obj/item/weapon/bananapeel,
+		/obj/item/weapon/soap,
+		/obj/item/weapon/bikehorn,
+		/obj/item/weapon/wrench,
+		/obj/item/weapon/wrench/socket,
```
Comment (ShiftyRail) [inline code/game/machinery/bots/secbot.dm]: If you're already checking for `wrench` this isn't needed, I think.

Comment (Arthurdentist) [reply code/game/machinery/bots/secbot.dm]: good catch

File: code/game/machinery/bots/secbot.dm [hunk 2]
```diff
@@ -1144,10 +1153,38 @@ Auto Patrol: []"},
...
+	else
+		new /obj/item/weapon/melee/classic_baton(Tsec)
+
+	if(prob(50))
+		new /obj/item/robot_parts/l_arm(Tsec)
+
+	spark(src)
+
+	var/obj/effect/decal/cleanable/blood/oil/O = getFromPool(/obj/effect/decal/cleanable/blood/oil, src.loc)
+	O.New(O.loc)
```
Comment (DamianX) [inline code/game/machinery/bots/secbot.dm]: Remove this line, `New` is already called by `getFromPool`

Comment (Arthurdentist) [reply code/game/machinery/bots/secbot.dm]: Removing that line just gives me an error, code\game\machinery\bots\secbot.dm:1182:warning: O: variable defined but not used

Comment (DamianX) [reply code/game/machinery/bots/secbot.dm]: You're supposed to remove 1184, the one with `New`.

Comment (Arthurdentist) [reply code/game/machinery/bots/secbot.dm]: I did remove it, and thats the error I get

Comment (DamianX) [reply code/game/machinery/bots/secbot.dm]: My bad, I misread the error (actually a warning). It's right, you don't need to declare `O` now that you're not manually calling its `New`, so change the line above to just `getFromPool(/obj/effect/decal/cleanable/blood/oil, src.loc)`, removing `var/obj/effect/decal/cleanable/blood/oil/O = `

Comment (Arthurdentist) [reply code/game/machinery/bots/secbot.dm]: alright, all sorted

File: code/game/machinery/bots/secbot.dm [hunk 3]
```diff
@@ -1104,3 +1116,129 @@ Auto Patrol: []"},
...
+			src.build_step++
+			to_chat(user, "You add the robot arm to [src]!")
+			src.name = "helmet/signaler/prox sensor/robot arm assembly"
+			src.overlays += image('icons/obj/aibots.dmi', "bhs_arm")
+			qdel(W)
+
+	else if((istype(W, /obj/item/weapon/melee/classic_baton)) && (src.build_step >= 3))
+		if(user.drop_item(W))
+			src.build_step++
+			to_chat(user, "You complete the Securitron! Beep boop.")
```
Comment (MadmanMartian) [inline code/game/machinery/bots/secbot.dm:1238]: It's not a securitron tho

File: code/game/machinery/bots/secbot.dm [hunk 4]
```diff
@@ -1180,9 +1179,8 @@ Auto Patrol: []"},
 
 	spark(src)
 
-	var/obj/effect/decal/cleanable/blood/oil/O = getFromPool(/obj/effect/decal/cleanable/blood/oil, src.loc)
-	O.New(O.loc)
-	qdel(src)
```
Comment (DamianX) [inline code/game/machinery/bots/secbot.dm]: why was this removed?
